code/game/objects/effects/spawners/windowspawner.dm

```diff
@@ -20,17 +20,17 @@
 			for(var/obj/effect/spawner/window/WS in get_step(src,cdir))
 				cdir = null
 				break
-			if(!cdir)	continue
+			if(!cdir)
+				continue
 			var/obj/structure/window/WI = new windowtospawn(get_turf(src))
 			WI.dir = cdir
 	else
-		var/obj/structure/window/W = new windowtospawn(get_turf(src))
-		W.dir = SOUTHWEST
+		new windowtospawn(get_turf(src))
 
 	if(useGrille)
 		new /obj/structure/grille(get_turf(src))
 
-	src.air_update_turf(1) //atmos can pass otherwise
+	air_update_turf(TRUE) //atmos can pass otherwise
 	// Give some time for nearby window spawners to initialize
 	spawn(10)
 		qdel(src)
```

Comment:
Right below here is the following code:
https://github.com/ParadiseSS13/Paradise/blob/fed65b1b02f59547c14e9bf225b8e772bc133da7/code/game/objects/effects/spawners/windowspawner.dm#L37-L38

Uncommenting that code (which was added [3 years ago](https://github.com/ParadiseSS13/Paradise/commit/b2d0bb1c0bd8b6216612fb536b3f499b5cfe5153)) and removing the `spawn(10)` block seems to function as expected still (on my local server).